content/en/docs/reference/access-authn-authz/kubelet-authn-authz.md

```diff
@@ -73,6 +73,23 @@ Kubelet API  | resource | subresource
 /spec/\*      | nodes    | spec
 *all others* | nodes    | proxy
 
+When the feature-gate `KubeletFineGrainedAuthz` is enabled kubelet performs a
```

Comment:
use the feature-gate embed here which will automatically pull the current feature state in and render it

Reply:
Done.